<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stu_change_password_</class>
 <widget class="QMainWindow" name="stu_change_password_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#stu_change_password_
{
	border-image: url(:/pic.png);
	background-color: rgb(255, 255, 255);
	
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="change_password_return">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QMainWindow#stu_change_password_ QPushButton#change_password_return
{
   font: 18pt &quot;宋体&quot;;
	
	background-color: rgb(255, 255, 255);
color: rgb(0,0,0);
}

QMainWindow#stu_change_password_ QPushButton#change_password_return:hover
{
	
	background-color: rgb(195, 195, 195);
}</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="change_password_new">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(214, 230, 255);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 12pt &quot;宋体&quot;;
color: rgb(0,0,0);</string>
    </property>
    <property name="placeholderText">
     <string>Places input new password</string>
    </property>
   </widget>
   <widget class="QLabel" name="change_password_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>修改密码界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_password_confirm">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>430</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QMainWindow#stu_change_password_ QPushButton#change_password_confirm
{
	
	background-color: rgb(174, 255, 186);
  color: rgb(0,0,0);

border:0px groove gray;border-radius:
7px;padding:2px 4px;
font: 18pt &quot;Candara&quot;;

}

QMainWindow#stu_change_password_ QPushButton#change_password_confirm:hover
{
	background-color: rgb(120,200,150);
}</string>
    </property>
    <property name="text">
     <string>确认修改</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
